### PR TITLE
[Doc] v5 Release Notes

### PR DESCRIPTION
**Pull request by furszy**
Initial cycle over the v5 release notes. 
Added information about the new introduced core features, block and transaction primitives changes, SHIELD GUI and wallet capabilities, tier two network improvements and the thread synchronization overhaul.

-- Pending proofreading --

### CODE REVIEW

**Comment by furszy** [comment]
Need proofreading, for me is ready to go.

File: doc/release-notes.md [hunk 1]
```diff
@@ -696,30 +696,282 @@ The table below details RPC commands that have now been fully removed. These com
...
+- #1895 `a418e9f1a3` [Bug][GUI] Fix wallet crashing on faq-buttons press (random-zebra)
+- #1900 `aafd0541c2` [QT] fixing qt flag deprecation (furszy)
+- #1919 `df2a6dbf60` [GUI] isStakingStatusActive pointers guarded. (furszy)
+- #1928 `15632533c3` [GUI] Double confirmation dialog for dumpwallet command. (furszy)
+- #1931 `65ca5e0255` [Bug][GUI] guarding wallet ptr access in WalletModel:getEncryptionStatus. (furszy)
+- #1963 `94c182e320` [Core] Sapling milestone 5: Graphical User Interface (furszy)
+- #1977 `bc2ec8590f` [GUI] Coin control dialog utxo processing/update flow large cleanup and improvements (furszy)
+- #1978 `f6a1a6e0e3` [GUI] FAQ "Exit" moved to "Close FAQ" (furszy)
+- #1997 `58f7562529` [Bug][GUI] CControl: cache utxo amount values properly (random-zebra)
+- #1999 `0805834485` [GUI] Encrypted memo feature implemented (random-zebra)
```
Comment (furszy) [inline doc/release-notes.md:717]: furszy

File: doc/release-notes.md [hunk 2]
```diff
@@ -696,30 +696,282 @@ The table below details RPC commands that have now been fully removed. These com
...
+- #2006 `9ce3fec43c` [Refactor] Masternode activeState on-demand (random-zebra)
+- #2023 `f9a29a9205` [Bug] Fix lock order issue with CMMan::CheckAndRemove and CMPayments::CleanPaymentList (random-zebra)
+- #2024 `a307a26eb6` [Bug] GetLastPaid/SecondsSincePayment locking cs_main (random-zebra)
+- #2031 `e7fcab0019` [Masternodes] remove recursive lock in CMasternode::Status() (random-zebra)
+- #2045 `1617f9d347` [TierTwo] Remove unneeded extra logging. (furszy)
+- #2056 `945044c740` [Masternode] Encapsulate global and do not parse privkey redundantly everywhere. (furszy)
+- #2060 `c246834912` [Bug][MN] Immediately detect forked masternodes (random-zebra)
+- #2068 `6db9e9c189` [Consensus] Budget: Fix instant killing of proposals (random-zebra)
+- #2084 `11606ee0d4` [BUG][Masternode] Miner not paying valid finalized budget if <20 active masternodes (random-zebra)
+- #2103 `7a1837eb58` [Startup] Do not block local peer port selection for Masternodes running over tor. (random-zebra)
```
Comment (furszy) [inline doc/release-notes.md:825]: furszy

**Review by random-zebra — APPROVED** [review]
ACK d88e6d89fdb33f378c04c0e92c9de5fd4dfeaabc

**Review by Fuzzbawls — APPROVED** [review]
ACK d88e6d89fdb33f378c04c0e92c9de5fd4dfeaabc

**Pull request merged**